i18n/zh/docusaurus-plugin-content-docs/current/contributor/lifted.md

```diff
@@ -1,39 +1,43 @@
 ---
-title: How to manage lifted codes
+title: 如何管理 Lift 代码
```

Comment:
Lift or Lifted？
What do you think is better?

Reply:
hmmm, this page has two forms of lift:
- lifted
- lifting
Less tenses are found in Chinese. In fact, I don't know how to translate `lift`, so just leave it as is. 😅

Reply:
Okay.